installer/main.sh

```diff
@@ -366,6 +368,15 @@ addtrap "stty echo 2>/dev/null"
 BIN="$PREFIX/bin"
 CHROOTS="$PREFIX/chroots"
 
+# Check if space requirements have been met
+# if not, check for dualboot
+if ! check_space "$SIZEMIN" "$AVAILMIN"; then
+    sleep 3
```

Comment:
Removed multiple delays and added message that installation will continue after 5s.